test/kevent.c

```diff
@@ -43,8 +43,10 @@ kevent_get(struct kevent *kev, int kqfd)
     struct kevent buf;
     int nfds;
 
-    if (kev == NULL)
-       kev = &buf;
+    if (kev == NULL) {
+        memset(&buf, 0, sizeof(buf));
```

Comment:
Why is this needed? The majority of applications won't initialise the kevent buffers before passing them in.  If there's uninitialised fields then kevent should be fixed to initialise them